### PR TITLE
Created screen for Upcoming Contests

### PR DESCRIPTION
**Pull request by imraghav20**

### CODE REVIEW

File: stopstalk_app/lib/screens/upcoming_contest_screen.dart [hunk 1]
```diff
@@ -1,23 +1,112 @@
...
+                  child: Text("Loading..."),
+                ),
+              );
+            } else {
+              return ListView.builder(
+                  itemCount: snapshot.data.length,
+                  itemBuilder: (BuildContext context, int index) {
+                    return Card(
+                      margin: EdgeInsets.only(
+                          left: 10.0, top: 5.0, right: 10.0, bottom: 5.0),
```
Comment (Jashpatel1) [inline stopstalk_app/lib/screens/upcoming_contest_screen.dart:70]: Also try some different left and right padding values and choose the best view.

Comment (imraghav20) [reply stopstalk_app/lib/screens/upcoming_contest_screen.dart:70]: Did the required changes.

File: README.md [hunk 2]
```diff
@@ -29,9 +29,11 @@ samples, guidance on mobile development, and a full API reference.
 3. Download an android emulator in the android studio or connect it to your phone(in debug mode) to run the app.
 3. When the app builds for the first time it might take a while for it to get launched.
 4. All the code that we have to work with is present in the lib folder.
-5. The screens folder contains the basic layout of each individual page. We add more widgets and make up the UI of the page.
-6. The widgets folder has individual widgets which can be used on multiple pages or are large in size and would reduce the readabilty and make a single page's code too long. 
-7. Before making a pull request, format the code with `Ctrl+Alt+L`. Add a comma after all the closing brackets as it will help the IDE in aligning the brackets and formatting the code. For more on this checkout [Code Formatting](https://flutter.dev/docs/development/tools/formatting) .
+5. To install any additionsl packages, first write its name under dependencies in file "pubspec.yaml". Then, run the command `flutter pub get`. The packages will be installed.
```
Comment (Jashpatel1) [inline README.md]: typo in "additional"

Comment (imraghav20) [reply README.md]: Resolved the typo.

File: stopstalk_app/lib/screens/contest_details.dart [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+import 'package:flutter/material.dart';
+
+import '../classes/contest_class.dart';
+
+class ContestPage extends StatelessWidget {
+  final Contest contest;
+
+  ContestPage(this.contest);
+
+  var images = {
```
Comment (Jashpatel1) [inline stopstalk_app/lib/screens/contest_details.dart]: var must be changed to final

File: stopstalk_app/lib/screens/upcoming_contest_screen.dart [hunk 4]
```diff
@@ -1,23 +1,114 @@
...
+          contest["Platform"],
+          contest["StartTime"],
+          contest["Duration"],
+          contest["EndTime"]);
+      contests.add(cont);
+    }
+    return contests;
+  }
+
+  var images = {
```
Comment (Jashpatel1) [inline stopstalk_app/lib/screens/upcoming_contest_screen.dart]: var must be changed to final

Comment (imraghav20) [reply stopstalk_app/lib/screens/upcoming_contest_screen.dart]: Made the required changes.

**Pull request merged**